api/src/main/resources/firecloud.yaml

```diff
@@ -264,6 +264,39 @@ paths:
           - openid
           - email
           - profile
+  /api/user/billing/{projectName}:
+    delete:
+      tags:
+        - Billing
+      summary: delete billing project
+      description: delete billing project
```

Comment:
Please elaborate on this, potentially for folks who haven't worked with the billing system or have forgotten what a billing project is.